nixos/modules/services/web-apps/rss-bridge.nix

```diff
@@ -98,7 +103,20 @@ in
         type = types.nullOr types.str;
         default = "rss-bridge";
         description = ''
-          Name of the nginx virtualhost to use and setup. If null, do not setup any virtualhost.
+          Name of the nginx or caddy virtualhost to use and setup. If null, do not setup any virtualhost.
+        '';
+      };
+
+      virtualHostType = mkOption {
```

Comment:
do we want this to be called `webserver`? same as e.g. `services.wordpress.webserver`

not really a blocking point, but consistency might be nice across nixpkgs.